<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Medical Photo Recognize</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="activationOrder">
       <enum>QMdiArea::CreationOrder</enum>
      </property>
      <property name="viewMode">
       <enum>QMdiArea::SubWindowView</enum>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>639</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionOpen_recognized_image"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuRecognize">
    <property name="title">
     <string>Recognize</string>
    </property>
    <widget class="QMenu" name="menuPoint_Recognize">
     <property name="title">
      <string>Point Recognize</string>
     </property>
     <addaction name="actionMaximum_minimum_distance_method"/>
     <addaction name="actionScreening_method"/>
    </widget>
    <widget class="QMenu" name="menuSquare_recognize">
     <property name="title">
      <string>Square recognize</string>
     </property>
     <addaction name="actionK_means_method"/>
     <addaction name="actionISODATA_method"/>
    </widget>
    <addaction name="menuPoint_Recognize"/>
    <addaction name="menuSquare_recognize"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuRecognize"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionMaximum_minimum_distance_method">
   <property name="text">
    <string>Maximum-minimum distance method</string>
   </property>
  </action>
  <action name="actionScreening_method">
   <property name="text">
    <string>Screening method</string>
   </property>
  </action>
  <action name="actionK_means_method">
   <property name="text">
    <string>K-means method</string>
   </property>
  </action>
  <action name="actionISODATA_method">
   <property name="text">
    <string>ISODATA method</string>
   </property>
  </action>
  <action name="actionOpen_recognized_image">
   <property name="text">
    <string>Open recognized image</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
